protocol/x/ratelimit/keeper/keeper.go

```diff
@@ -130,8 +127,8 @@ func (k Keeper) GetBaseline(
 	limiter types.Limiter,
 ) *big.Int {
 	// Get the current TVL.
-	// TODO(CORE-836): Query bank Keeper to get current supply of the token.
-	currentTVL := TempTVLPlacerholder
+	supply := k.bankKeeper.GetSupply(ctx, denom)
+	currentTVL := supply.Amount.BigInt()
```

Comment:
I don't think this comment is relevant since both the current and new CosmosSDK versions won't return errors. In case of internal error, the `GetSupply` returns a 0 coin. 

Of course, panic is a different story

Old: https://github.com/dydxprotocol/cosmos-sdk/blob/bdf96fddf421e90574e53af383c1468fdb33d7a3/x/bank/keeper/keeper.go#L218

New: https://github.com/dydxprotocol/cosmos-sdk/blob/b48250d4e0efacbd40f3e0b980c1e1004d29e351/x/bank/keeper/keeper.go#L201